src/encryption/symmetric/README.md

```diff
@@ -0,0 +1,40 @@
+# Symmetric Encryption algorithms
+
+[Symmetric encryption](https://en.wikipedia.org/wiki/Symmetric-key_algorithm) algorithms are cryptographic algorithms that uses same key for encryption and decryption.
+
+## Block Ciphers
+
+TODO
+
+## Stream ciphers
+
+[stream ciphers](https://en.wikipedia.org/wiki/Stream_cipher) are symmetric encryption cryptography primitives that works on digits, often bits, rather than fixed-size blocks as in [block-ciphers](https://en.wikipedia.org/wiki/Block_cipher).
+
+$$
+\begin{align*}
+\text{Enc}(s,m)=c=G(s)\oplus m \\
```

Comment:
i have written FAR too much TeX... but another option instead of `\text{}` is `\operatorname{}`

Don't worry about doing that here, but it can help you out some point in your life I bet.

Or hopefully we never see TeX again in a few years...

Reply:
![8vizys](https://github.com/pluto/ronkathon/assets/35560828/c38483bb-4b2c-47ce-8054-b1d1f2e314f1)